app/views/devise/registrations/new.haml

```diff
@@ -6,14 +6,18 @@
         .card-body
           %h1= t('views.sessions.new')
           = bootstrap_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
-            = devise_error_messages!
+            = render 'devise/shared/error_messages', resource: resource
```

Comment:
I think this change is no longer required if the shared error messages template hasn't changed from upstream